NetimobiledeviceDemo/Program.cs

```diff
@@ -38,6 +39,10 @@ internal static async Task Main()
 
         await Task.Delay(1000);
 
+        using (LockdownClient lockdown = LockdownClient.CreateLockdownClient(testDevice?.Serial ?? string.Empty)) {
+            Console.WriteLine($"Current connected model : {ModelIdentifier.GetDeviceModelName(lockdown.Product)}");
```

Comment:
I was thinking the way we would want to use this would be as a part of LockdownClient. So make the ModelIdentifier class internal and then in LockdownClient create a public property called ProductFriendlyName or similar and have that get the model and use the ModelIdentifier.GetDeviceModelName function.

Reply:
Made the changes as stated.